chef/automate
# CDS download

### :nut_and_bolt: Description: What code changed, and why?

When the "download" button is clicked on a content item that item will be downloaded to the browser. This is streaming the file from a third-party source and streaming it to the user's browser. 

### :chains: Related Resources

#3899 

### :athletic_shoe: How to Build and Test the Change
1. `build components/automate-gateway && build components/automate-cds && build components/automate-ui-devproxy && start_all_services && start_automate_ui_background && ui_logs`
1. Go to https://a2-dev.test/cds
1. Click download on the three different content items listed and ensure that they download and have different file names. (The filename will be different but they all download this file https://github.com/dev-sec/apache-baseline/archive/master.tar.gz)

### :white_check_mark: Checklist
